### ISSUE TITLE
Error while creating new user

### ISSUE THREAD

**Issue by mmitucha** [getsentry/sentry#1529]
Hello,

I just ran into problem while **creating a new user**.  
When I submit the form with **Create project** checked, I get this:
```
[ERROR] Internal Server Error: /manage/users/new/
Traceback (most recent call last):
  File "/home/www/python/sentry/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 112, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/www/python/sentry/local/lib/python2.7/site-packages/sentry/web/decorators.py", line 236, in wrapped
    return func(request, *args, **kwargs)
  File "/home/www/python/sentry/local/lib/python2.7/site-packages/sentry/web/decorators.py", line 245, in wrapped
    return func(request, *args, **kwargs)
  File "/home/www/python/sentry/local/lib/python2.7/site-packages/django/db/transaction.py", line 371, in inner
    return func(*args, **kwargs)
  File "/home/www/python/sentry/local/lib/python2.7/site-packages/django/utils/decorators.py", line 99, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "/home/www/python/sentry/local/lib/python2.7/site-packages/sentry/web/frontend/admin.py", line 137, in create_new_user
    name='%s\'s New Project' % user.username.capitalize()
  File "/home/www/python/sentry/local/lib/python2.7/site-packages/django/db/models/manager.py", line 157, in create
    return self.get_queryset().create(**kwargs)
  File "/home/www/python/sentry/local/lib/python2.7/site-packages/django/db/models/query.py", line 322, in create
    obj.save(force_insert=True, using=self.db)
  File "/home/www/python/sentry/lib/python2.7/site-packages/sentry/models/project.py", line 119, in save
    slugify_instance(self, self.name, organization=self.organization)
  File "/home/www/python/sentry/local/lib/python2.7/site-packages/django/db/models/fields/related.py", line 326, in __get__
    "%s has no %s." % (self.field.model.__name__, self.field.name))
DoesNotExist: Project has no organization.
```

I tried to look into [create_new_user](https://github.com/getsentry/sentry/blob/2f0f366c9ca2c965f7776194854a9b8efef68988/src/sentry/web/frontend/admin.py#L115-L173) function and [project](https://github.com/getsentry/sentry/blob/2f0f366c9ca2c965f7776194854a9b8efef68988/src/sentry/models/project.py) model, but could not find any Organization assigment, while new user is created. Even when I try to create new 'Project' from shell like [here](https://github.com/getsentry/sentry/blob/2f0f366c9ca2c965f7776194854a9b8efef68988/src/sentry/web/frontend/admin.py#L136-L138), I get the same Exception.
Don't know how it should work, if the user is assigned to default 'Sentry' Organization, or if new one is created for him.

I even thought I could have made something wrong, while creating Projects and Organizations before. 
So I flushed all Redis instances, dropped database and created new one clean. 
But problem persists.

The version of Sentry is v7.4.3 on Ubuntu 14.04(LTS) installed from pypi in virtualenv according to [install-sentry](http://sentry.readthedocs.org/en/latest/quickstart/index.html#install-sentry).

I would be very grateful for any advice.

**Comment by dcramer** [comment]
You can use /admin/ to do this as a workaround. The other admin pages we haven't put a lot of care into and they're not super functional. That's something we'll be changing in the future.

**Comment by mmitucha** [comment]
Thank you very much for your answer. 

Using admin is functional workaround for sure, but I would like to not force other admins to use Django Generic Admin interface.

I can easily fix this for my needs, but I would like the know, how the mechanism of creating new user works. If it should be assigned to default 'Sentry' organization, or if new one should be created for him.

**Comment by dcramer** [comment]
@mmitucha we're aiming to solve this more with SSO, but I think it depends

If its under SENTRY_SINGLE_ORGANIZATION mode it should bind them to the existing organization, if not I don't think it should do anything.

**Comment by mmitucha** [comment]
I was able to fix it for our needs in stable (7.4.4) version, where 'SENTRY_SINGLE_ORGANIZATION' does not exists, but works great.
Thank you very much for help.
